prime-router/settings/organizations.yml

```diff
@@ -987,4 +987,15 @@
         host: sftp
         port: 22
         filePath: ./upload
-        credentialName: DEFAULT-SFTP
\ No newline at end of file
+        credentialName: DEFAULT-SFTP
+- name: "CDC-ELIMS-HL7"
```

Comment:
Optional, but could add a testing receiver here as well and add an additional assertion in [this](https://github.com/CDCgov/prime-reportstream/blob/251110fb97cca7f5662a9e3b21af7f0c01b18428/prime-router/src/main/kotlin/cli/tests/BasicTests.kt#L91) smoke test

Reply:
A receiver here overlaps a bit with #10200, but I think that would be fine (especially since I'll need it in [my PR](https://github.com/CDCgov/prime-reportstream/pull/10114) once this is merged), and then that ticket can be more focused on the transport side of things.

Reply:
I added to the end2end_up test which required adding a sender and receiver to the ignore org. So with that I deleted the CDC ELIMS org and sender that I had previously added